rla/proto/v1/rla.proto

```
@@ -448,29 +448,26 @@ message ValidateComponentsResponse {
     
     // Summary counts
-    int32 only_in_expected_count = 3;
-    int32 only_in_actual_count = 4;
+    int32 missing_count = 3;     // Expected in local DB but missing from source system
+    int32 unexpected_count = 4;  // Found in source system but not expected in local DB
     int32 drift_count = 5;
     int32 match_count = 6;
 }
 
 enum DiffType {
     DIFF_TYPE_UNKNOWN = 0;
```

Buf (1.67.0)

[error] Enum zero value name "DIFF_TYPE_UNKNOWN" should be suffixed with "_UNSPECIFIED".

(ENUM_ZERO_VALUE_SUFFIX)